runner/src/rpc/api.rs

```diff
@@ -97,7 +95,7 @@ pub async fn view(
     method_name: String,
     args: FunctionArgs,
 ) -> Result<serde_json::Value, String> {
```

Comment:
Can we swap this to be generic on serde deserialize to not have to go through serde_json::Value if a caller doesn't want? (prob in another PR)

Reply:
yea definitely. A lot of the RPC calls need to be re-evaluated too. I'll make a ticket for another PR later once this merges